rpcserver.go

```diff
@@ -1283,16 +1287,25 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 
 	var txid *chainhash.Hash
 
-	wallet := r.server.cc.Wallet
+	lnWallet := r.server.cc.Wallet
 	maxFeeRate := r.cfg.Sweeper.MaxFeeRate.FeePerKWeight()
 
+	var selectOutpoints []wire.OutPoint
```

Comment:
should move this block to where it is used, down in the else branch. To make this easier to read we might want to refactor the if/else into two separate methods.